modular_zubbers/code/modules/antagonists/traitor/uplink/uplink_items/syndimaid/clothes.dm

```diff
@@ -16,44 +16,34 @@
 	armor_type = /datum/armor/clothing_under/syndimaid
 	body_parts_covered = CHEST|GROIN|LEGS
 	resistance_flags = parent_type::resistance_flags | FIRE_PROOF
+	/// who we are currently locking from wearing other outer clothing
+	var/datum/weakref/wear_locked
```

Comment:
wear_locked just exists so we don't use a boolean state here which could potentially be fucky

Reply:
we *really* don't want dropped after-effects to run on not the original equipped target (fucky stuff happens)